client/packages/android/app/src/main/java/org/openmsupply/client/NativeApi.java

```diff
@@ -327,26 +330,25 @@ private NsdServiceInfo createLocalServiceInfo() {
         serviceInfo.setAttribute(discoveryConstants.PROTOCOL_KEY, "https");
         serviceInfo.setAttribute(discoveryConstants.CLIENT_VERSION_KEY, "unspecified");
         serviceInfo.setAttribute(discoveryConstants.HARDWARE_ID_KEY, discoveryConstants.hardwareId);
-       return serviceInfo;
+        return serviceInfo;
     }
 
     // Had issues resolving local server when wifi and mobile data is off
-    // getting onResolveFailed with errorCode = 0 (internal error) with no more information
+    // getting onResolveFailed with errorCode = 0 (internal error) with no more
+    // information
     // manually adding this server should work
     private void addLocalServerToDiscovery() {
-        if(isAdvertising && isDiscovering && discoveredServers != null) {
+        if (isDiscovering && discoveredServers != null) {
```

Comment:
So when we run as client we `do` want local server to be discovered ?

Reply:
I have reverted that - I thought that it was required, but cannot see why now after re-reading the code. will run a few more tests, but my view is that we don't need local when running in client mode.